server/app/services/member-service.js

```diff
@@ -205,6 +207,30 @@ async function deleteMember(memberId) {
   return member;
 }
 
+/**
+ * Search members which match query parameter in all database.
+ *
+ * @param query {string} query.
+ * @return {Promise<Member[]>} Members which match the query
+ */
+async function searchMembers(query) {
+  logger.verbose('Member service: searching members with query %s', query);
+  const where = Sequelize.where(
+    Sequelize.fn('concat', Sequelize.col('firstName'), ' ', Sequelize.col('lastName')),
+    'like',
+    `%${query}%`,
+  );
```

Comment:
To do search queries, we should probably use https://www.w3resource.com/mysql/mysql-full-text-search-functions.php in order to optimize queries

Reply:
@kistora We will do it later, it working for now (with +3000 members)